content/20.treatments.md

```diff
@@ -224,17 +224,23 @@ Summarize the state of the antiviral approach.
 
 #### Hydroxychloroquine and Chloroquine
 
-Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and disease-modifying anti-rheumatic drugs in humans.
-HCQ and CQ act through the same mechanism of action, but HCQ is regarded to be safer [@doi:10.1136/ard.2008.101766].
-HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
-_In vitro_ studies of CQ use for SARS-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi:10.1186/1743-422X-2-69].
-In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
-A case study of three COVID-19 patients reported presence of antiphospholipid antibodies in all three patients [@doi:10.1056/NEJMc2007575].
-Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome. 
-HCQ has been reported to be effective in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023; @doi:10.1016/j.autrev.2014.01.053; @doi:10.1182/asheducation-2016.1.714].
-
-Thus, there is excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
-This section will discuss current efforts on assessing the potential impact of HCQ in treating COVID-19.
+CQ and HCQ are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
+Both drugs increase cellular pH by accummulating in their protonated form inside lysosomes [@doi:10.1186/1743-422X-8-163; @doi:10.1111/j.1529-8019.2007.00131.x].
+This shift in pH inhibits the breakdown of proteins and peptides by the lysosomes during the process of proteolysis [@doi:10.1111/j.1529-8019.2007.00131.x].
+A number of mechanisms have been proposed through which these drugs could influence the immune response to pathogen challenge.
+For example, CQ/HCQ can interfere with digestion of antigens within the lysosome and inhibit CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells.
```

Comment:
I'm not sure if you want to reference the same paper again - it seems to be coming from `10.1111/j.1529-8019.2007.00131.x`. Since the pattern has been to reference it in the other sentences, I added it here.
```suggestion
For example, CQ/HCQ can interfere with digestion of antigens within the lysosome and inhibit CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells [@doi:10.1111/j.1529-8019.2007.00131.x].
```